llama_stack/core/library_client.py

```diff
@@ -298,9 +325,6 @@ async def request(
         stream=False,
         stream_cls=None,
     ):
-        if self.route_impls is None:
```

Comment:
how is this related?

Reply:
since we removed `initialize()` and we have `_auto_initialize()`